Classes/View Controllers/UIViewController+Safari.swift

```diff
@@ -12,8 +12,12 @@ import SafariServices
 extension UIViewController {
 
     func presentSafari(url: URL) {
-		guard let safariViewController = try? SFSafariViewController.configured(with: url) else { return }
-        route_present(to: safariViewController)
+        if UserDefaults.standard.shouldOpenExternalLinksInSafari {
+            UIApplication.shared.open(url)
+        } else {
+            guard let safariViewController = try? SFSafariViewController.configured(with: url) else { return }
+            route_present(to: safariViewController)
```

Comment:
@rnystrom: why is this using snake case? 😮

Reply:
experimenting w/ prefix for a utils extension

Reply:
@rnystrom Should I fix it?

Reply:
Not a big deal right now IMO